pkgs/tools/networking/ipfetch/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitHub, bash, wget, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "ipfetch";
+  version = "unstable-2022-03-24";
+
+  src = fetchFromGitHub {
+    owner = "trakBan";
+    repo = "ipfetch";
+    rev = "fc295bfda4f9fea6eee9f6f3f2dabc26b6f25be4";
+    sha256 = "sha256-YKQ9pRBj2hgPg2ShCqWGxzHs/n7kNhKRNyElRDwHDBU=";
+  };
+
+  strictDeps = true;
+  buildInputs = [ bash wget ];
+  nativeBuildInputs = [ makeWrapper ];
+  patchPhase = ''
```

Comment:
Please don't overwrite patchPhase, use postPatch instead.

Reply:
I pushed the fix to the repo now. Should I open another PR? I mean it does work, after all.

Reply:
Yes, please open another PR. It works as long as you don't use patches.